src/store/AppInfo.ts

```diff
@@ -50,8 +51,8 @@ const appInfoState: AppInfoState = {
     loadingDisableCloseButton: false,
     hasControlsDisabled: false,
     controlsDisabledMessage: '',
-    faucetUrl: networkConfig.faucetUrl,
-    settings: settingService.getProfileSettings(ANON_PROFILE_NAME),
+    faucetUrl: networkConfig[NetworkType.TEST_NET].faucetUrl,
```

Comment:
why hardcoded here? should read from current profile perhaps?

Reply:
Since the Faucet won't be available in mainnet, I didn't work on tying it to the profile network type. I can work on it if it doesn't make sense.

Reply:
Is it a big change to use profile network type? In all I think it doesn't makes sense to have networktype hardcoded anymore

Reply:
fixed